### ISSUE TITLE
Issue with Hotstar and JioTv

### ISSUE THREAD

**Issue by upwan111** [botallen/repository.botallen#68]
Hi,

I have installed Kodi Leia 18.7 on my laptop and installed JioTV and Hotstar add on. It was working perfectly fine for few days.
It stopped working suddenly. I am not sure what has changed.

I tried uninstalling then re-installing fresh with add-on but still same issue is happening.

I have attached the logs for the reference. Can you please look into it?

Thanks.
[kodi.log](https://github.com/botallen/repository.botallen/files/4939192/kodi.log)

**Comment by iMro0t** [comment]
@upwan111  are you using any sort of VPN ? Looks like you are trying to access from another region from log. Both add-on will now work with VPN. So i would suggest don't use VPN

**Comment by upwan111** [comment]
Yes, I am VPN to change the location to India. It is working perfectly fine with on my mobile and firestick though. I checked once again now on my laptop. I am able to watch using VPN on it. 
Strange thing now I see, all the videos are coming in black and white format. Not sure, why.
